packages/devextreme/js/__internal/ui/stepper/stepper.ts

```diff
@@ -302,6 +302,16 @@ class Stepper extends CollectionWidgetAsync<StepperProperties> {
     }
   }
 
+  _hover($el?: dxElementWrapper, $previous?: dxElementWrapper): void {
+    const $hoveredStep = this._findHoverTarget($el)?.get(0);
```

Comment:
```suggestion
    const hoveredStepElement = this._findHoverTarget($el)?.get(0);
```
_shouldPreventItemEvent method requires only Element as argument or we can pass dxElementWrapper there directly?

Reply:
discussed, fixed